app/static/js/home/home.js

```diff
@@ -13,6 +13,16 @@ $(function() {
         }, 200  );
     });
 
+    // Email regex
+    var pattern = new RegExp(/^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$/i);
```

Comment:
this is used to validate whether or not the value of the input is a valid email address, so that we can animate the "sign up" button into appearance